### DISCUSSION TITLE
Guide: How to install Hwinfo Plugin

### DISCUSSION THREAD

**Discussion by Fr0stX76** [Rem0o/FanControl.Releases#714] [General]
First, download the file from [here](https://ci.appveyor.com/project/Rem0o/fancontrol-hwinfo/branch/master/artifacts)

While fancontrol is closed, unzip the files from the zip in the plugin folder of Fancontrol, if needed, unblock the dll (see [here](https://2.bp.blogspot.com/-PZjsMvjdLf8/VzpHwD95KAI/AAAAAAAABwA/XZ3v-bbCc7or7I2ObLiRVK2kyyFZFJkKQCLcB/s640/security.png)), start fancontrol, activate the plugin in the sensor menu (top right 3 dots), refresh sensors (you might need to restart FanControl).

Voila

Note: This plugin rely on a feature of HWinfo called "Shared memory support". You will need to activate it in HWinfo (settings, General/ user interface tab). Furthermore, unless you use the paying version of HWinfo (PRO) the feature will deactivate itself every 12hrs. This has nothing to do with FanControl as it is a limitation of HWinfo free version.

Note2:  This plugin rely on the order of the sensors listed in HWinfo, meaning if the order of sensor changes in HWinfo (ex. if a sensor is added, removed or hidden in HWinfo while running), FanControl will returrn an incorrect or NaN reading and the fans associated with that sensor will ramp up or stop. This will happen if the added or removed sensor is over (on top) of the one FanControl is reading from in HWinfo list. I found out that using a custom order of sensor in HWinfo and moving the sensor I read from at the very top of the list will prevent this from happening.

**Comment by archlord2k3** [comment]
do u need to have  HWinfo memory share open?? while fan controller is open?
thx for the details step by step 
i was ussing hwmonitor thinking it was the same thing lol

**Reply by Fr0stX76** [reply]
You need HWinfo to run at all time but It can be minimized to tray. And yes the "shared memory support" option of HWinfo need to be activated or else it will stop "sharing" sensors to other software (like FanControl). FYI, the option in HWinfo is located in the settings, under the General/user inteface tab

**Reply by archlord2k3** [reply]
yea i got that 
thx!!
wasnt sure if it needed to run the other program to
i have a AMD cpu should I be using the core TCT/TDIE for the temp graph
I cant find cpu package my guess its the same thing

**Reply by Fr0stX76** [reply]
I'm not sure, since I'm on Intel I've always used the temp of my highest core. But you might want to read [this](https://www.hwinfo.com/forum/threads/cpu-temp-sensors-explanation.5597/#post-31437)
